server/jobs/index.js

```diff
@@ -40,6 +47,49 @@ const jobs = {
   },
 };
 
+/**
+ * @function sendPlacementFetchEmail
+ * @desc Send email to ESA if fetching placements fails constantly
+ * @param {string} receiver Info about the mail to be sent
+ *
+ * @returns {Object} Promise to fetch new placements and execute automations
+ */
+const sendPlacementFetchEmail = async (receiver) => {
```

Comment:
Can you move this function to `server/jobs/helpers.js`

Reply:
The function has been moved